test_box_easy.js

```diff
@@ -0,0 +1,122 @@
+/* eslint-disable camelcase */
```

Comment:
Once this has been finished, this test file should be moved to a new PR against https://github.com/sodium-friends/sodium-test